auth-web/src/views/auth/IncorpOrRegisterView.vue

```diff
@@ -8,7 +8,7 @@
           <v-icon size="8" class="list-item-bullet mt-5">mdi-square</v-icon>
           <v-list-item-content>
             <v-list-item-subtitle class="list-item-text">
-              If you have an approved Name Request (NR number), or you want a <NumberedCompanyTooltip />,
+              If you have an approved Name Request (NR number), or you want a numbered Benefit Company,
```

Comment:
hope you removed the component intentionally

Reply:
Yes indeed :) This tooltip was just giving some template text, which we fully intend to use in the future but the definition verbiage was still pending so i removed it for the time being, in the event that we don't have the verbiage by the time the MVP is released :)